css/style.css

```diff
@@ -59,18 +61,178 @@ body {
 
 /* Center the container */
 #container {
-  width: 960px;
-  margin: auto;
+  min-width: 700px;
+  padding:20px 50px;
+  box-sizing: border-box;
+}
+
+header h1 {
+  font-size: 24px;
+  font-weight: 800;
+  color: white;
+  display: inline;
 }
 
+nav a {
+  color: #f9e42e;
+  text-decoration: none;
+  font-size: 24px;
+  padding-left: 10px;
+  font-weight: 600;
+  text-align: right;
+ }
+
+ nav {
+  float: right;
+ }
+
+ nav a:hover {
+  font-style: italic;
+
+ }
+
+
 header {
+  background-color: #033048;
+  padding-left: 60px;
+  padding-right: 60px;
+  padding-top: 35px;
+  padding-bottom: 35px;
+  width: auto; 
+}
+ h2 {
+  font-size: 40px;
+  font-weight: 600;
+  color: #033048;
+  padding: 0;
+ }
+
+ section p {
+  font-size: 24px;
+ }
+
+ .firstcharacter{
+  color: #606161;
+  float: left;
+  font-size: 75px;
+  line-height: 60px;
+  padding-top: 4px;
+  padding-right: 8px;
+  padding-left: 3px;
+}
 
+section nav a {
+  font-size: 24px;
+  color: #033048;
+  text-decoration: none;
+  display:block;
 }
 
-footer {
+.solid-bottom {
+  border-bottom: 2px solid #033048;
 
 }
 
+h3{
+  font-weight: 800;
+  font-size: 32px;
+  margin: 0;
+}
+
+li {
+  font-size: 24px;
+  font-weight: 300;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding-left: 0px;
+    padding-bottom:10px;
+    text-align: left;
+
+}
+
+aside p {
+  font-size: 22px;
+  font-weight: 300;
+}
+
+aside a {
+  font-size: 22px;
+  font-weight: 800;
+  color: #033048;
+  text-decoration: none;
+}
+
+aside a:hover{
+  font-style: italic;
+}
+
+
+.call-to-sign {
+  text-align: center;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  background-color: #f9e42e;
+
+}
+
+.mastfooter {
+  background-color: #121212;  
+  text-align: center;
+  padding: 5%;
+}
+
+.copyright {
+  font-size: 12px;
+  font-weight: bold;
+  color:#f0efef;
+  padding: 10px;
+}
+
+.media-links {
+  display: inline-block;
+  box-sizing: border-box;
+}
+
+footer img {
+  height: 10%;
+  width: 10%;
+  padding: 5px;
+}
+
+footer a {
+  text-decoration: none;
+}
+
+.call-to-style {
+  text-decoration: none;
+  color: #f9e42e;
+  border-radius: 5px;
+  font-weight: bold;
+  background-color: #033048;
+  padding: 5px 20px;
+  font-size: 12px;
+}
+
+section {
+  max-width: 700px;
+  overflow: hidden;
+}
+
+aside {
+  width: 25%;
+  float: left;
+  box-sizing: border-box;
+  padding-top: 20px;
+}
+
+main {
+  width: 75%;
+  box-sizing: border-box;
```

Comment:
rather than apply these everywhere, I recommend just doing:

```css
html {
 font-size: border-box;
}

*, *::after, *::before {
  box-sizing: inherit;
}
```